app/screens/map-screen/map-screen.tsx

```diff
@@ -131,7 +131,7 @@ export const MapScreen: ScreenType = ({ navigation }: Props) => {
         ? navigation.navigate("sendBitcoin", { username: item.username })
         : navigation.navigate("phoneValidation")
     markers.push(
-      <Marker coordinate={item.coordinate} key={item.title}>
+      <Marker coordinate={item.coordinate} key={item.title} pinColor={'#FF8000'}>
```

Comment:
We may also want to extract this into https://github.com/GaloyMoney/galoy-mobile/blob/main/app/theme/palette.ts . Or use the orange from that file.